machines/openId4VP/openId4VPActions.ts

```diff
@@ -0,0 +1,207 @@
+import {assign} from 'xstate';
+import {send, sendParent} from 'xstate/lib/actions';
+import {SHOW_FACE_AUTH_CONSENT_SHARE_FLOW} from '../../shared/constants';
+import {VC} from '../VerifiableCredential/VCMetaMachine/vc';
+import {StoreEvents} from '../store';
+
+import {VCShareFlowType} from '../../shared/Utils';
+
+let predefinedPresentationDefinitions = {
```

Comment:
why we need this?

Reply:
Added it for supporting the scope request param (alias for Presentation definition) of Authorization request. In code we have already added if else conditions for supporting the scope param so we didn't want to remove it and we can use this logic to show case that scope param is supported in any call. So we have created a dummy variable for this purpose. I will add a TODO comment in the code saying we will get this data from verifier endpoint.

Reply:
Please add a comment in code